cpu/saml21/Makefile.features

```diff
@@ -1 +1,6 @@
--include $(RIOTCPU)/sam0_common/Makefile.features
+# samr30 does not contain a TRNG
+ifeq (,$(findstring samr30,$(CPU_MODEL)))
+  FEATURES_PROVIDED += periph_hwrng
+endif
+
+include $(RIOTCPU)/sam0_common/Makefile.features
```

Comment:
This line should be a different commit/PR as it needs its specific review.

Nice find, I missed it when reviewing not included 'common' features.

Reply:
That line is not new, I just changed `-include` to `include` to be consistent with the rest of sam0.

Reply:
Then please split it in another commit.

Reply:
Ok, I split it to another commit.